apps/tangent-electron/src/app/views/creation-rules/CreationRuleEditor.svelte

```diff
@@ -154,4 +159,21 @@ figcaption {
 	}
 }
 
+textarea { /* Add style for textarea */
```

Comment:
How many of these rules are actually required to get the desired effect? We may want to provide a default setup in input.scss.

The comments here are not really necessary.

Also, spaces instead of tabs.